context-py/src/lib.rs

```diff
@@ -362,6 +361,42 @@ fn associate_codeowners_multithreaded_impl(
     Ok(results)
 }
 
+#[gen_stub_pyfunction]
+#[pyfunction]
+#[pyo3(signature = (
```

Comment:
This feels wrong to me? This came out of what looks like there being too many `Option<String>`s